cumulusci/cli/logger.py

```diff
@@ -14,6 +23,9 @@ def init_logger(log_requests=False):
     for handler in logger.handlers:  # pragma: nocover
         logger.removeHandler(handler)
 
+    if os.name == "nt" and "colorama" in sys.modules:
```

Comment:
I think this is another reasonable place to use `# pragma: nocover` to avoid reporting an uncovered line.